src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  @for $index from 1 through 5 {
```

Comment:
@for loop should be nested to &__star element

Reply:
that`s okay, because I use `&--#{$index} > &__star:nth-child(-n + #{$index})` , so I don`t need to put in, I think